src/Mod/Part/Gui/TaskCheckGeometry.cpp

```diff
@@ -149,7 +149,7 @@ QString checkStatusToString(const int &index)
     }
     if (index > 33 || index < 0)
     {
-        QString message(QObject::tr("Out Of Enum Range: "));
+        QString message(QObject::tr("Out Of Enum Range:") + QString::fromLatin1(" "));
```

Comment:
IMHO if you touch not only string contents but also code itself, you should not skip CI

Reply:
fair enough